src/handlers/vestingschedule.ts

```diff
@@ -25,13 +28,15 @@ export class VestingScheduleHandler {
   }
 
   public async save () {
+    logger.debug('handleVestingScheduleAddedEvent event data: '  + JSON.stringify(this.data.toHuman()))
```

Comment:
Is `JSON.stringify` needed here?

Reply:
Yes, if not it will be printed as `[Object, object]`

Reply:
I would then doubt if `toHuman` is doing a useful conversion there. You know what I mean?

Reply:
toHuman fn is working as we expect, but the logger is not so the JSON.stringify is a workaround for that